File: SortArrayByParity.py
class Solution:
    def sortArrayByParity(self, A):
        """
        :type A: List[int]
        :rtype: List[int]
        """
        arr = []
        n = len(A)
        for i in range(n):
            if(A[i] % 2 != 0):
                arr.append(A[i])
            else:
                arr.insert(0,A[i])
        return arr

File: SortArrayByParity.js
var sortArrayByParity = function(A) {
    var arr = [];
    for(var i=0;i<arr.length;i++){
        if(A[i] % 2 != 0){
            arr.push(A[i]);
        }
        else{
            arr.unshift(A[i]);
        }
    }
    console.log(arr)
};
var A = [3,1,2,4];
sortArrayByParity(A)